### ISSUE TITLE
Allow alternative style for printed commands

### ISSUE THREAD

**Issue by bew** [casey/just#1041]
### Example run:
![image](https://user-images.githubusercontent.com/9730330/144355996-8444e50d-a1d3-4909-8887-bbb674bf7091.png)

:warning: **The bold style is too subtle**, it's a bit hard to distinguish the printed command from its output..
I'd prefer to have them is an entirely different color, or even with a prefix like:
```
>> Running: some-command arg1 arg2
```

:point_right: Maybe this could be configured with a format string?
`set command_format_string := ">> Running: {}"`
_(there I can put ANSI escape sequence for colors if I want)_

What do you think?

**Comment by casey** [comment]
I think this should probably be configurable with an environment variable or a config file. My thinking is that this is a user-specific preference, and not something that should be configured in a justfile.

How about `JUST_COMMAND_ECHO_PREFIX` and `JUST_COMMAND_ECHO_SUFFIX` environment variables for configuration?

**Comment by bew** [comment]
> How about `JUST_COMMAND_ECHO_PREFIX` and `JUST_COMMAND_ECHO_SUFFIX` environment variables for configuration?

Yeah you're right, it's definitely a user config, env vars is enough for me :+1:

**Comment by casey** [comment]
This can be configured with `--command-color`.
